msu/load.nut

```diff
@@ -29,3 +29,6 @@ includeLoad("hooks");
 includeLoad("msu_mod");
 includeLoad("vanilla_mod");
 includeLoad("test");
+
+this.Const.World.RoadBrushes.add(::Const.Direction.N, "road_N");
+this.Const.World.RoadBrushes.add(::Const.Direction.S, "road_S"); // idk where to put these
```

Comment:
That is easy. Put them in `msu/config/world.nut` as that is the folder where we keep all our changes to vanilla `scripts/config` files.